Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.yml

```diff
@@ -0,0 +1,114 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Google Safe Browsing v2
+  version: -1
+configuration:
+- defaultvalue: https://safebrowsing.googleapis.com/v4/threatMatches:find
+  display: Your server URL
+  name: url
+  required: true
+  type: 0
+- additionalinfo: The API Key to use for connection
+  display: API Key
+  name: api_key
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: The Client ID
+  hidden: false
+  name: client_id
+  required: false
+  type: 0
+- defaultvalue: '1.0'
+  display: The Client Version
+  hidden: false
+  name: client_version
+  required: false
+  type: 0
```

Comment:
this should be right below the server_url param. order is:
connection params
thresholds/reliability
sslproxy

Reply:
done